pkgs/development/python-modules/pianotrans/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, piano-transcription-inference
+, pytorch
+, tkinter
+, ffmpeg
+}:
+
+buildPythonPackage rec {
```

Comment:
This should probably be `buildPythonApplication` and live outside of `python-modules` and instead live in `pkgs/applications` somewhere.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function

Reply:
Done.